### PR TITLE
Reduce Jupyter image size

### PR DESCRIPTION
**Pull request by Yancey0623**
Remove some unused dependencies, reduce the image size from 2.14GB to 894MB.

### CODE REVIEW

File: docker/jupyter/Dockerfile [hunk 1]
```diff
@@ -16,15 +18,12 @@ COPY docker/jupyter/js /jupyter/js
 
 RUN apt-get -qq update
 
-COPY /docker/ci/install-python.bash /jupyter
-RUN /jupyter/install-python.bash
-
-COPY docker/ci/install-pips.bash /jupyter/
-RUN /jupyter/install-pips.bash
```
Comment (lhw362950217) [inline docker/jupyter/Dockerfile:23]: If we don't install pips, we can't use `numpy` and others in `Jupyter Notebook`, are we intend to do so?

File: docker/jupyter/Dockerfile [hunk 2]
```diff
@@ -16,15 +18,12 @@ COPY docker/jupyter/js /jupyter/js
...
-
 COPY docker/jupyter/install-jupyter.bash /jupyter
 RUN /jupyter/install-jupyter.bash
 
+# back to the default user
+USER jovyan
```
Comment (lhw362950217) [inline docker/jupyter/Dockerfile]: How comes this user?

File: docker/jupyter/Dockerfile (hunk 1 above)
Comment (Yancey0623) [reply docker/jupyter/Dockerfile:23]: Maybe users can install the Python tools using `pip install` in Jupyter Notebook easily.

File: docker/jupyter/Dockerfile (hunk 2 above)
Comment (Yancey0623) [reply docker/jupyter/Dockerfile]: I added some comments here, please check it again.

**Review by lhw362950217 — APPROVED** [review]
LGTM

**Pull request merged**